src/resources/SecurityCache/SecurityCache.ts

```diff
@@ -36,7 +36,11 @@ export default class SecurityCache extends Ressource {
     }
 
     refreshCache() {
-        return this.api.post(`${SecurityCache.cacheUrl}/refresh`);
+        return this.api.post(`${SecurityCache.cacheUrl}/refresh`, {});
```

Comment:
You can omit the `{}` now
```suggestion
        return this.api.post(`${SecurityCache.cacheUrl}/refresh`);
```

Reply:
Lets omit it :
`_omit( theCall, "{}")`
:+1: thanks